src/main/interface/msp_box.h

```diff
@@ -25,6 +25,8 @@ typedef struct box_s {
     const uint8_t permanentId;      // permanent ID used to identify BOX. This ID is unique for one function, DO NOT REUSE IT
 } box_t;
 
+#define PERMANENT_ID_INVALID 255
```

Comment:
Not sure, maybe `PERMANENT_ID_NONE` ?

Reply:
Good idea. I've also moved corresponding `BOXID_NONE` to `rc_modes.h`.